Examples/Idealgas.py

```diff
@@ -0,0 +1,131 @@
+from openmmtools.testsystems import IdealGas
+from openmm import unit
+
+
+# Use the IdealGas example from openmmtools to initialize particle positions and topology
+# For this example, the topology provides the masses for the particles
+
+n_particles = 216
```

Comment:
216 = 6**3.  Since everything is non-interacting the system can be very small.  I used 216 as that was the default in the openmm-tools idealgas test system.